src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -26,6 +26,13 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 
 	try {
 		const url = new URL(`${req.protocol}://${req.get("host")}${req.originalUrl}`);
+		if (!servers.hasCloudServer && !servers.gheServerInfos.length) {
```

Comment:
This is seperate logic to what the try block is doing, we can move it just below 
`const servers = await getGitHubServers(jiraHost);`

Reply:
Moved...